src/compute/src/render/mod.rs

```diff
@@ -526,6 +526,12 @@ where
             Some(ArrangementFlavor::Local(oks, errs)) => {
                 // Set up probes to notify on index frontier advancement.
                 oks.stream.probe_notify_with(probes);
+
+                // Attach logging of dataflow errors.
+                if let Some(logger) = compute_state.compute_logger.clone() {
+                    errs.stream.log_dataflow_errors(logger, idx_id);
```

Comment:
This will cause all errors to be cloned. Better assign the output to `errs` to avoid attaching multiple operators to the `errs` stream.

Reply:
We are not using `errs.stream` anymore (and we can't since we are moving out of it here) so there shouldn't be any cloning happening, right?